From: Seth Heasley <seth.heasley@intel.com>
Date: Mon, 23 Apr 2012 09:23:56 -0700
Subject: [PATCH] pci_ids: Add Intel Centerton Legacy Block DeviceID
Patch-mainline: 1fc9b1eade80b323f02a9cf7a29e1641eddf1052
References: FATE#313512

This patch adds the Integrated Legacy Block DeviceID for the Centerton CPU.  It
will be used in the GPIO and Multifunction Devices driver.

Signed-off-by: Seth Heasley <seth.heasley@intel.com>
Signed-off-by: Samuel Ortiz <sameo@linux.intel.com>
Acked-by: Jiri Kosina <jkosina@suse.cz>
---
 include/linux/pci_ids.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/linux/pci_ids.h b/include/linux/pci_ids.h
index 3329965..ab741b0 100644
--- a/include/linux/pci_ids.h
+++ b/include/linux/pci_ids.h
@@ -2506,6 +2506,7 @@
 #define PCI_DEVICE_ID_INTEL_MRST_SD2	0x084F
 #define PCI_DEVICE_ID_INTEL_I960	0x0960
 #define PCI_DEVICE_ID_INTEL_I960RM	0x0962
+#define PCI_DEVICE_ID_INTEL_CENTERTON_ILB	0x0c60
 #define PCI_DEVICE_ID_INTEL_8257X_SOL	0x1062
 #define PCI_DEVICE_ID_INTEL_82573E_SOL	0x1085
 #define PCI_DEVICE_ID_INTEL_82573L_SOL	0x108F
-- 
1.7.8.3

